src/components/colorscale/color_attributes.js

```diff
@@ -35,14 +35,19 @@ module.exports = function makeColorScaleAttributes(context) {
                 ' values are required. For example,',
                 ' `[[0, \'rgb(0,0,255)\', [1, \'rgb(255,0,0)\']]`.',
                 ' To control the bounds of the colorscale in color space,',
-                ' use `', context, '.cmin` and `', context, '.cmax`.'
+                ' use `', context, '.cmin` and `', context, '.cmax`.',
+                ' Alternatively, `colorscale` may be a palette name string',
+                ' such as `"Viridis"` or `"Greens"`.'
             ].join('')
         }),
         cauto: extendDeep({}, colorScaleAttributes.zauto, {
             description: [
-                'Has an effect only if `', context, '.color` is set to a numerical array.',
-                ' Determines the whether or not the color domain is computed',
-                ' automatically.'
+                'Has an effect only if `', context, '.color` is set to a numerical array',
+                ' and `cmin`, `cmax` are set by the user. In this case,',
```

Comment:
@etpinard in line with your above notes on the `cmin`/`cmax` auto-calculation, I think my text addition is legit: if the user does *not* set `cmin`/`cmax` then they'll be auto-calculated to the `color` array lo/hi values, IOW `cauto` will have no (visible) effect.

Sorry to be so tedious about the doc strings but I'm adding some test cases in general to reify the official API behaviors and since it's nontrivial stuff (and in some cases, by necessity, there are semiarbitrary choices), I feel the need to work against definitions that are clear to me. E.g. if you want to permit that only one of `cmin`/`cmax` is specified, I'd not only update the docs but also make tests that way so it's not purely about the doc strings.